policyengine_us/variables/gov/states/mi/tax/income/deductions/is_social_security_retirement.py

```diff
@@ -0,0 +1,9 @@
+from policyengine_us.model_api import *
+
+
+class is_social_security_retirement(Variable):
```

Comment:
This belongs in the main income variable section, and as a float which you can check exceeds zero. But first check that this isn't already in the system.